pkg/apiserver/domain/service/workflow.go

```diff
@@ -43,6 +43,7 @@ import (
 	"github.com/oam-dev/kubevela/pkg/oam/util"
 	utils2 "github.com/oam-dev/kubevela/pkg/utils"
 	"github.com/oam-dev/kubevela/pkg/utils/apply"
+	"github.com/oam-dev/kubevela/references/cli"
 )
```

Comment:
Strongly discourage importing reference/ package in pkg/ package. Related functions should be moved in pkg/.

Reply:
exactly yes!

Reply:
maybe we will migrate the cli into a separate project some day